index.html

```diff
@@ -133,16 +133,19 @@ <h4>Skills</h4>
                 <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```

Comment:
[ OPTIONAL ] Please add all unnecessary files and folders like `.vscode` to `.gitignore`

Reply:
Hi @SajjadAhmad14 

Thank you so much for the optional suggestion, I'll do the needful. I appreciate it.